yarn.lock

```diff
@@ -11133,6 +11135,15 @@ d3-force@1:
     d3-quadtree "1"
     d3-timer "1"
 
+d3-force@^2.1.1:
```

Comment:
It is kind of weird that you only added d3-force and yet there are many more changes here. Probably not going to do harm.

Reply:
hmm true, bit weird, I had some other libs here when testing then removed, afterwards got some conflicts with master. I think I will try to reset this to master and install d3-force on clean lock file.

Reply:
looks better 👌